tools/openapi-plugin/src/generators/client/generator.ts

```diff
@@ -26,6 +28,9 @@ export async function clientGenerator(
         );
         contents = await getLocalSchema(tree.root, schemaPath);
     }
+
+    await validate(schemaPath);
+
     tree.write(`${projectRoot}/types/index.d.ts`, contents);
```

Comment:
I can see that we're using `${projectRoot}/types/index.d.ts` here and also in `convertIntegerTypes()` function. Would it be better to:
1. Pass the file pat to `convertIntegerTypes()` as a param so we don't have to hardcode it twice?
2. Atm, we write the content to the Tree and then read it again from file system to do some transformation. Can we instead do the transformation first then write it to the Tree later?
Eg:
```typescript
// Assuming the function definition is:
// export function convertIntegerTypes(filePath: string, tree: Tree, content: string) {}
// Then we call it like so:
convertIntegerTypes(`${projectRoot}/types/index.d.ts`, tree, content);
```

Reply:
Good call :+1: